docs/doc/15-sql-functions/60-conversion-functions/index.md

```diff
@@ -32,5 +32,5 @@ Databend also offers a variety of functions for converting expressions into diff
 | TO_UINT16( expr )             | Converts a value to UINT16 data type                                   | TO_UINT16('123')                                    | 123                        |
 | TO_UINT32( expr )             | Converts a value to UINT32 data type                                   | TO_UINT32('123')                                    | 123                        |
 | TO_UINT64( expr )             | Converts a value to UINT64 data type                                   | TO_UINT64('123')                                    | 123                        |
-
+| TO_VARIANT(bitmap)          | Converts a bitmap value to VARIANT data type | TO_VARIANT(TO_BITMAP('100,200,300')) | [100,200,300] |
```

Comment:
`to_variant` is not a new function, the pr just fixed a bug cast from bitmap, other types expr can also cast to variant.

Reply:
As your suggestion is to indicate in to_variant support bitmap?

Reply:
Thank you. I just tested the following statements, and they worked fine:

SELECT to_variant(to_bitmap('100,200,300'));
SELECT to_variant(10);
SELECT to_variant('abcdefghijklmn');
SELECT to_variant(10.001);

I updated again the function as follows:
TO_VARIANT( expr )            | Converts a value to VARIANT data type